lib/iex/resources/ohlc.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'ohlc/base'
+
+module IEX
+  module Resources
+    module OHLC
+      def self.get(symbol)
+        resource = IEX::Api::OHLC.get(symbol)
+        OHLC::Base.new resource
+      rescue Faraday::ResourceNotFound => e
+        raise IEX::Errors::SymbolNotFoundError.new(symbol, e.response[:body])
+      end
+
+      def self.market
+        response = IEX::Api::OHLC.market
+        response.each do |key, value|
+          response[key] = OHLC::Base.new(value)
+        end
```

Comment:
You should probably create another hash instead of modifying the existing one, I am going to guess it's faster and cleaner, so:

```ruby
response = Hash[IEX::Api::OHLC.market.map do |k, v|
   [k, OHLC::Base.new(v)]
end]
```